applications/FluidDynamicsApplication/custom_processes/distance_modification_process.cpp

```diff
@@ -106,6 +108,8 @@ void DistanceModificationProcess::CheckDefaultsAndProcessSettings(Parameters &rP
     mAvoidAlmostEmptyElements = rParameters["avoid_almost_empty_elements"].GetBool();
     mNegElemDeactivation = rParameters["deactivate_full_negative_elements"].GetBool();
     mRecoverOriginalDistance = rParameters["recover_original_distance_at_each_step"].GetBool();
+    mDoubleVariablesList = rParameters["double_variables_list"].GetStringArray();
+    mComponentVariablesList = rParameters["component_variables_list"].GetStringArray();
```

Comment:
I would suggest initializing vectors of variables (or references/pointers of variables) from the string-based input. This allows you to validate the input at construction (if KratosComponents does not have the variable, you will know it here) and minimizes (expensive) calls to KratosComponents<...>::Get().